packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -30,22 +30,9 @@
     get-link-tokens-from-bg($bg-color, $preferred-link-token: $selected-color),
     1
   );
-  $input-disabled-alpha: -0.7;
-  $input-disabled-alpha--strong: -0.5;
-  $input-disabled-alpha--weak: -0.8;
   $input-border-alpha: -0;
-  $color-input-disabled: color.adjust(
-    color($input-text-color),
-    $alpha: $input-disabled-alpha
-  );
-  $color-input-disabled--strong: color.adjust(
-    color($input-text-color),
-    $alpha: $input-disabled-alpha--strong
-  );
-  $color-input-disabled--weak: color.adjust(
-    color($input-text-color),
-    $alpha: $input-disabled-alpha--weak
-  );
```

Comment:
Removed alpha adjustments in favor of new theme color settings.

Reply:
Big fan of cleaning this up as well!